public/index.html

```diff
@@ -45,6 +45,74 @@ <h3 class="oxford location-heading">Oxford</h3>
             </div>
           </div>
 
+          <div class="job">
+            <p><a class="job-title">Paid Web Developer Internship</a></p>
+            <div class="job-description">
+              <p>
+                The Zooniverse is excited to offer a paid 3-month, web developer
+                internship. Ideally you will be able to travel to our office in
+                Oxford (UK), however we will consider you if you want to work
+                remotely or part-time.
+
+                The internship will run over summer 2017 with the option for flexible
+                start and end dates.
+              </p>
+              <p>
+                There will be opportunities to work on a variety of projects
+                during the internship, according to our current needs as well as
+                your own skills and interests.
+
+                Some possible projects may include:
+                <ul>
+                  <li>Adding new front-end features to our site</li>
+                  <li>Working on new back-end API services</li>
+                  <li>Building data visualization tools</li>
+                  <li>Extracting services from our API</li>
+                  <li>Operations work with Docker and AWS</li>
+                  <li>A project you dream up and propose?</li>
+                </ul>
+              </p>
+              <p>
+                You should apply if you wish to be a web developer and you desire
+                to gain experience with any (not all) of the following:
+                <ul>
+                  <li>HTML</li>
+                  <li>CSS</li>
+                  <li>JavaScript</li>
+                  <li>Ruby</li>
+                  <li>Python</li>
+                  <li>Git</li>
+                  <li>Bash</li>
+                  <li>Docker</li>
+                  <li>AWS</li>
+                </ul>
+              </p>
+              <p>
+                We are an equal opportunity employer and value diversity at the
+                Zooniverse that does not discriminate but is committed to a
+                diverse and inclusive work environment. We particularly welcome
+                applications those with a background that is under-represented
+                in the technology sector.
+
+                This internship would suit current students, recent graduates,
+                those wishing for a career change or dreaming to return from a break.
+              </p>
+              <p>
+                The salary is at Grade 07S.01
+                <a href="https://www1.admin.ox.ac.uk/finance/epp/payroll/scales/grades6andup/">
+                  scale
+                </a>,
+                pro-rata. Applications will be accepted until we make a decision
+                in June, 2017.
```

Comment:
This needs a formal date to be set that leaves long enough to get applications, 2 weeks?